sdk/eventhub/azure-eventhub/azure/eventhub/_configuration.py

```diff
@@ -30,14 +31,20 @@ def __init__(self, **kwargs):
         self.max_batch_size = kwargs.get("max_batch_size", self.prefetch)  # type: int
         self.receive_timeout = kwargs.get("receive_timeout", 0)  # type: int
         self.send_timeout = kwargs.get("send_timeout", 60)  # type: int
-        self.custom_endpoint_address = kwargs.get("custom_endpoint_address")  # type: Optional[str]
+        self.custom_endpoint_address = kwargs.pop(
```

Comment:
so for this one, I'm actually popping it b/c we modify the custom endpoint address in config, and then pass that new one in to different places. However, if we use get here, we'll get an error that says we're passing it in twice. I'm popping this to avoid that error.

Reply:
jk just double checked and it looks like this doesn't actually affect anything. good catch! changing this back.